cfme/services/requests.py

```diff
@@ -2,13 +2,21 @@
 from contextlib import contextmanager
 from cfme.exceptions import RequestException
 from cfme.fixtures import pytest_selenium as sel
-from cfme.web_ui import Region, SplitTable, fill, flash, paginator, toolbar, Input
+from cfme.web_ui import Region, SplitTable, fill, flash, paginator, Table, toolbar, Input
+from utils import deferred_verpick, version
 from utils.log import logger
 
-request_list = SplitTable(
-    ('//*[@id="list_grid"]//table[contains(@class, "hdr")]/tbody', 1),
-    ('//*[@id="list_grid"]//table[contains(@class, "obj")]/tbody', 1)
-)
+request_list = version.pick({
```

Comment:
It would be much wiser to wrap the `request_list` in a `Region` since `Region` automatically verpicks the dictionaries. This `version.pick` verpicks on module load which is not good.